docbrown/core/src/tpartition.rs

```diff
@@ -184,7 +217,7 @@ mod temporal_graph_partition_test {
 
         let expected_len = vs.iter().map(|(v, _)| v).sorted().dedup().count();
         for (v, t) in vs {
-            g.add_vertex(t.into(), v.into(), &vec![]);
+            g.add_vertex(v.into(), t.into(), &vec![]);
```

Comment:
@miratepuffin I think we want `t` to be the first argument in the `add_` methods?

Reply:
Some places it was first arg while other it was second, so thought of keeping it consistent